README.md

```diff
@@ -54,7 +54,7 @@ params = Parameters(simulation = SimulationParameters(working_dir=working_dir,
 		    hyper = Hyperparameters(batch_size=4,
 					    epochs=10,
 					    optimizer=ODINN.ADAM(0.01)),
-		    UDE = UDEparameters(target = "A")
+		    UDE = UDEparameters(target = :A)
```

Comment:
As mentioned in #302, the idea would be to map targets with functions directly in the new `Law` types which would be fed into iceflow and mass balance models.